docs/pre-release/releaseNotes.yml

```diff
@@ -30,6 +30,35 @@ docDescription: >-
 changelog: https://github.com/telepresenceio/telepresence/blob/$branch$/CHANGELOG.md
 
 items:
+  - version: 2.3.4
+    date: '2021-07-09'
+    notes:
+      - type: bugfix
+        title: Improved IP log statements
+        body: >-
+          Fixed log statements that were printing incorrect IP addresses.
+      - type: bugfix
+        title: Improved messaging when multiple services match a workload
+        body: >-
+          If multiple services matched a workload when performing an intercept, Telepresence would panic.
```

Comment:
Nit: I prefer to write "crash" instead of "panic".  IMO it conveys the meaning better to those who aren't familiar with Go.